sdcm/cluster.py

```diff
@@ -1408,23 +1408,26 @@ def config_setup(self, seed_address=None, cluster_name=None, enable_exp=True, en
         if server_encrypt or client_encrypt:
             self.remoter.send_files(src='./data_dir/ssl_conf',
                                     dst='/tmp/')
-            self.remoter.run('sudo mv /tmp/ssl_conf/*.* /etc/scylla/')
+            self.remoter.run('mkdir -p ~/.cassandra/')
```

Comment:
same code is used in `node_setup`. Why?

Reply:
Thats' the node_setup of loader.
We run c-s on loader,  run cqlsh & c-s on db node.